.github/workflows/ci.yml

```diff
@@ -49,12 +49,15 @@ jobs:
         with:
           node-version: '16.x'
           cache: 'yarn'
+      - run: |
+          echo "CGW_VERSION=`npm pkg get version`" >> $GITHUB_ENV
```

Comment:
This is not validated in our tests right? If not I'd consider removing it until we need it.

Reply:
Yes, removed :) [9c26798](https://github.com/5afe/safe-client-gateway-nest/pull/190/commits/9c2679898364e7474e96937607672fd3b810cacb)